### DISCUSSION TITLE
New DAB Radio installation fails

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1195] [Bug Reports]
Purchased: Nooelec NESDR Mini 2 USB RTL-SDR & ADS-B Receiver Set, RTL2832U & R820T2 Tuner

Added DABRadio addon to RPI3A rAudio-1 previously updated to 20230117
Rebooted RPi

Went to Features and enabled DAB Radio

Clicked on Info icon and got:

rtsp-simple-server 0.21.1-1
# rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T/2 tuner
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6 
Sampling at 2048000 S/s.
Testing tuner range. This may take a couple of minutes..

● rtsp-simple-server.service
     Loaded: loaded (/usr/lib/systemd/system/rtsp-simple-server.service; enabled; preset: disabled)
     Active: active (running) since Thu 2023-01-19 16:13:52 GMT; 1min 30s ago
   Main PID: 5012 (rtsp-simple-ser)
      Tasks: 8 (limit: 459)
        CPU: 84ms
     CGroup: /system.slice/rtsp-simple-server.service
             └─5012 /usr/bin/rtsp-simple-server /etc/rtsp-simple-server/rtsp-simple-server.yml

Jan 19 16:13:52 rAudio systemd[1]: Started rtsp-simple-server.service.
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF rtsp-simple-server v0.0.0
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [RTMP] listener opened on :1935
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [HLS] listener opened on :8888
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [WebRTC] listener opened on :8889 (HTTP)

Go Library / DAB Radio / Scan Digital Radio Stations - OK
Wait a while, nothing apparently happens

Go back to Features / DAB Radio
DAB Radio now disabled

Clicked on Info icon and got:

rtsp-simple-server 0.21.1-1
# rtl_test -t
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
usb_claim_interface error -6
Failed to open rtlsdr device #0.

○ rtsp-simple-server.service
     Loaded: loaded (/usr/lib/systemd/system/rtsp-simple-server.service; disabled; preset: disabled)
     Active: inactive (dead)

Jan 19 16:11:27 rAudio systemd[1]: Stopped rtsp-simple-server.service.
Jan 19 16:13:52 rAudio systemd[1]: Started rtsp-simple-server.service.
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF rtsp-simple-server v0.0.0
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [RTMP] listener opened on :1935
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [HLS] listener opened on :8888
Jan 19 16:13:52 rAudio rtsp-simple-server[5012]: 2023/01/19 16:13:52 INF [WebRTC] listener opened on :8889 (HTTP)
Jan 19 16:20:31 rAudio systemd[1]: Stopping rtsp-simple-server.service...
Jan 19 16:20:31 rAudio systemd[1]: rtsp-simple-server.service: Deactivated successfully.
Jan 19 16:20:31 rAudio systemd[1]: Stopped rtsp-simple-server.service.

Subsequent attempts to enable DAB Radio result in: No DAB devices found.

Disconnect USB device and reconnect allows DAB Radio Function to be enabled as at the top of this post. 

What am I missing to get this device connected and usable.

Thanks for any help anybody can provide.

Chris

**Comment by hjfisch** [comment]
Exactly the same for me
